webapp/channels/src/components/emoji/emoji_page.tsx

```diff
@@ -25,62 +25,52 @@ type Props = {
     };
 }
 
-export default class EmojiPage extends React.PureComponent<Props> {
-    static defaultProps = {
-        teamName: '',
-        teamDisplayName: '',
-        siteName: '',
-    };
-
-    componentDidMount() {
-        this.updateTitle();
-        this.props.actions.loadRolesIfNeeded(['system_admin', 'team_admin', 'system_user', 'team_user']);
+export default function EmojiPage({teamDisplayName = '', teamName = '', siteName = '', scrollToTop, currentTheme, actions}: Props) {
```

Comment:
Can we have every prop in a new line?
```suggestion
export default function EmojiPage({
    teamDisplayName = '',
    teamName = '',
    siteName = '',
    scrollToTop,
    currentTheme,
    actions,
}: Props) {
```